arviz/plots/backends/matplotlib/elpdplot.py

```diff
@@ -138,6 +140,8 @@ def plot_elpd(
                 sharey="row",
                 sharex="all",
             )
+        else:
+            fig = ax.get_figure()
```

Comment:
I think it should be `ax.ravel()[0]` or something like this, because in these cases `ax` is an array of axes, they should all have the same figure as reference but the array does not know about the figure.